engine/src/main/resources/default.cfg

```diff
@@ -24,6 +24,7 @@
     "windowWidth": 1152,
     "windowHeight": 720,
     "fullscreen": false,
+    "windowedFullscreen": false,
```

Comment:
Should this be merged with `fullscreen` and turned into enum values instead of booleans?